src/dotnet-restore/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "dotnet-restore",
+  "description": "Restore packages specified in project.json (project file)",
+  "version": "1.0.0-*",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/dotnet/cli"
+  },
+  "compilationOptions": {
+    "warningsAsErrors": true,
+    "emitEntryPoint": true
+  },
+  "dependencies": {
```

Comment:
The multiproject validator will catch this as well since you are creating mismatched versions with the other projects